13-TeV-examples/uproot_python/ttbar_analysis.ipynb

```diff
@@ -0,0 +1,1236 @@
+        " $t\\bar{t}$ analysis code provided:\n",
+        "\n",
+        "1.   **Integrated Luminosity (lumi)**\n",
+        "\n",
+        "  *   Definition: Integrated luminosity is a measure of the total amount of data collected by a particle detector over a certain period. It represents the total number of potential collisions that could have occurred in a particle accelerator and is typically measured in inverse femtobarns fb$^{-1}$.\n",
+        "\n",
+        "2. **Fraction of Events to Process**\n",
+        "\n",
+        "  *   Definition: This parameter controls what fraction of the available events in the dataset will be processed by the analysis.\n",
+        "\n",
+        "3. **Conversion Factor from MeV to GeV**\n",
+        "\n",
+        "  * Definition: Particle physics often uses energy units like MeV (Mega electron Volts) and GeV (Giga electron Volts). The conversion factor helps switch between these units.\n",
+        "\n",
+        "4. **Minimum Transverse Momentum $p_𝑇$ Limit Definition**\n",
+        "  * Transverse momentum $p_𝑇$ is a crucial quantity in particle physics, representing the momentum of particles perpendicular to the beam axis. It's often used as a selection criterion for events.\n",
+        "\n",
+        "5. **MV2c10 b-Tagging Algorithm Discriminant Cut Value Definition**\n",
+        "  * The MV2c10 is a b-tagging algorithm used to identify jets originating from b-quarks. The discriminant cut value determines the threshold for tagging a jet as a b-jet.\n",
+        "\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "NeWGPZrCeWTT"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Integrated luminosity in inverse femtobarns\n",
+        "lumi = 10000.\n",
+        "\n",
+        "# Fraction of events to process (1 means all events)\n",
+        "fraction = 1\n",
+        "\n",
+        "# Conversion factor from MeV to GeV\n",
+        "MeV2GeV = 0.001\n",
+        "\n",
+        "# Minimum transverse momentum (pT) limit in GeV\n",
+        "pt_lim = 30\n",
+        "\n",
+        "# MV2c10 b-tagging algorithm discriminant cut value\n",
+        "MV2c10_lim = 0.828"
+      ],
+      "metadata": {
+        "id": "rMdNPb42cNA5"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# **Samples**\n",
+        "Samples to process"
+      ],
+      "metadata": {
+        "id": "u6CY1t7X1YzH"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Dictionary of samples to be processed\n",
+        "samples = {\n",
+        "    # Real data samples\n",
+        "    'data': {'list' : ['data_A','data_B','data_C','data_D'],},\n",
+        "\n",
+        "    # Main Monte Carlo (MC) sample: top quark pair production\n",
+        "    'MC' : {  'list' : ['mc_410000.ttbar_lep'],},\n",
```

Comment:
The `MC` nomenclature might me a bit misleading: is only this sample MonteCarlo? Then what are SingleTop and Diboson? I suggest we either rename MC somthing like "ttbar", or we add a MC prefix to all the MC samples.